examples/wind/src/layers/delaunay-cover-layer/delaunay-cover-layer.js

```diff
@@ -92,9 +92,10 @@ export default class DelaunayCoverLayer extends Layer {
         setParameters(gl, {
           blend: true,
           blendFunc: [gl.SRC_ALPHA, gl.ONE_MINUS_SRC_ALPHA],
-          blendEquation: gl.FUNC_ADD
+          blendEquation: gl.FUNC_ADD,
+          depthTest: true,
```

Comment:
So there was a depth test issue after all... It has bit me in the past as well, annoying that we lose so much time on this. Maybe we should turn on depth testing by default after all. Maybe open an issue on this so we can discuss? There was a suggestion to change default blending as well today...